backend/routes/users.js

```diff
@@ -50,11 +50,27 @@ users.get('/:id', async ctx => {
 });
 
 users.patch('/:id', parsers.json, async (ctx) => {
-    try{ctx.body = await Users.query()
-        .patch({isVerified: true})
-        .findById(ctx.params.id)}
+    let response = {}
+    try{
+        let targetUser = await Users.query()
+        .findById(ctx.params.id)
+
+        //currently, we have to have dateRegistered and lastLogin in patch request or it won't work
+        let patchedPayload = Object.assign(ctx.request.body, {dateRegistered: targetUser.dateRegistered, lastLogin: 0})
+
+        //actual patch request
+        await Users.query()
+        .patch(patchedPayload)
+        .findById(ctx.params.id)
+
+        response.result = 'User object successfuly modified';
+        ctx.body = JSON.stringify(response);
+    }
     catch (err){
-        ctx.body = err
+        response.result = 'Could not modify user object';
```

Comment:
I have a better idea about this as well, than I had last time. :) Sorry for the iterative thinking. We should do this:

response = {
 status: 'error'
 errorText: 'Could not modify user object'
 originalError: err
 }